internal/gapicgen/generator/config.go

```diff
@@ -1524,5 +1530,8 @@ var MicrogenGapicConfigs = []*MicrogenConfig{
 		// GAPIC-level retries and allow the veneer layer to handle retries.
 		ApiServiceConfigPath: "storage_v2.yaml",
 		ReleaseLevel:         "alpha",
+		// Results in protobuf/grpc code generated into the following package:
+		// cloud.google.com/go/storage/internal/apiv2/stubs.
+		StubsDir: "stubs",
```

Comment:
Should this have internal in here?

Reply:
The implementation actually uses this as a suffix on the ImportPath, which in this case is `cloud.google.com/go/storage/internal/apiv2` - it already has internal in it.